pkg/Microsoft.NETFramework.Compatibility/Microsoft.NETFramework.Compatibility.pkgproj

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <PackageVersion>2.1.0</PackageVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <LibraryPackage Include="System.CodeDom" />
+    <LibraryPackage Include="System.ComponentModel.Annotations" />
+    <LibraryPackage Include="System.Composition" />
+    <LibraryPackage Include="System.Composition.AttributedModel" />
+    <LibraryPackage Include="System.Composition.Convention" />
+    <LibraryPackage Include="System.Composition.Hosting" />
+    <LibraryPackage Include="System.Composition.Runtime" />
+    <LibraryPackage Include="System.Composition.TypedParts" />
+    <LibraryPackage Include="System.Configuration.ConfigurationManager" />
+    <LibraryPackage Include="System.Data.SqlClient" />
+    <LibraryPackage Include="System.DirectoryServices" />
```

Comment:
@tquerec what is the quality level of each of these? 
System.DirectoryServices
System.DirectoryServices.AccountManagement
System.DirectoryServices.Protocols